lib/metanorma/cli/site_generator.rb

```diff
@@ -121,30 +143,20 @@ def config_from_manifest
         end
       end
 
-      def manifest_config(manifest)
+      def manifest_config(manifest_from_yaml)
         {
-          files: extract_config_data(
-            manifest["metanorma"]["source"], "files"
-          ) || [],
+          files: manifest_from_yaml.dig("metanorma", "source", "files") || [],
```

Comment:
@ribose-jeffreylau perhaps replace this with lutaml-model instead of wrestling further with YAML?

Reply:
Could do.  Then next step is to model the `metanorma.yml` file in LutaML.

Reply:
I've pasted sample code below. Hope this helps!